community/modules/compute/gke-node-pool/README.md

```diff
@@ -28,6 +28,91 @@ can be overridden using the `taints` setting. See
 [docs](https://cloud.google.com/kubernetes-engine/docs/how-to/node-taints) for
 more info.
 
+### Considerations with GPUs
+
+When a GPU is attached to a node an additinal taint is automatically added:
+`nvidia.com/gpu=present:NoSchedule`. For jobs to get placed on these nodes the
+equivalent toleration is required. When using the `gke-job-template` module this
+toleration will automatically be applied when using a node pool with GPUs.
+
+Nvidia GPU drivers must be installed by applying a DaemonSet to the cluster. See
+[these instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#cos).
+
+### GPUs Examples
+
+There are several ways to add GPUs to a GKE node pool. See
+[docs](https://cloud.google.com/compute/docs/gpus) for more info on GPUs.
+
+The following is a node pool that uses `a2` or `g2` machine types which has a
+fixed number of attached GPUs:
+
+```yaml
+  - id: simple-a2-pool
+    source: community/modules/compute/gke-node-pool
+    use: [gke_cluster]
+    settings:
+      machine_type: a2-highgpu-1g
+```
+
+> **Note**: It is not necessary to define the [`guest_accelerator`] setting as
```

Comment:
I would clarify that this is only true for a2/g2 VM families. (I think this is your intent)

Reply:
Done.